VR-Alfama/Assets/Scripts/TrialSession.cs

```diff
@@ -155,12 +172,49 @@ IEnumerator Session()
             LogPointToMap(2, FinalWorldPoint); // TODO - Do we need to log this before the user has made a firm choice on map position?
 
             // Reset
+            UiManager.CloseImagePanel();
             CurrentTrialIndex++;
         }
+        LogSession(1);
+        UiManager.OpenMessagePanel("Session complete", "Please wait for someone to take the headset.");
+        while (true) 
+        yield return null;
+    }
 
-        UiManager.OpenMessagePanel("Session complete", "Please wait for someone to take the headset.", 1f);
+    private void LogGeoreference()
```

Comment:
```suggestion
    private void LogGeoReference()
```

For consistency with property name and type.

Reply:
Fixed in f7b7cdf7c8fcfd58473671cb87b3e598aa1d2e31